multicast_group: '224.0.0.1'
multicast_port: 9000
server_address: '192.168.1.101'
buffer_size: 2048

# Rigid Body Names
rigid_body_names:
  - Robot_1
